src/L.Control.Locate.js

```diff
@@ -104,12 +108,19 @@ You can find the project at: https://github.com/domoritz/leaflet-locatecontrol
             },
             /** Inner marker style properties. Only works if your marker class supports `setStyle`. */
             markerStyle: {
-                color: '#136AEC',
+                color: '#fff',
                 fillColor: '#2A93EE',
-                fillOpacity: 0.7,
-                weight: 2,
-                opacity: 0.9,
-                radius: 5
+                fillOpacity: 1,
+                weight: 3,
+                opacity: 1,
+                radius: 7
+            },
+            /** Compass */
+            compassStyle: {
```

Comment:
Let's make the compass a bit larger or somehow easier to see (maybe white outline?? could be ugly).

<img width="220" alt="screen shot 2018-11-05 at 07 58 30" src="https://user-images.githubusercontent.com/589034/48009460-a15f5900-e0d0-11e8-9b61-8a058f3c90f9.png">

Reply:
Maybe shadow? Do you have an idea?

Reply:
Maybe we can add some shadow to the circle, make the circle slightly larger and move the direction indicator closer to the blue circle. It might make it easier to see the direction.

Reply:
The new style for the marker is fairly similar to both the apple maps and google maps apps (on both web and ios at least), and the arrow is based of the apple maps arrow. Google maps has a kinda blue 'tunnel' showing where you are looking which I don't think is as obvious. I don't think this really needs to be much more largest or higher contrast because when you turn around it's clearly moving against the map background and your eye jumps to it easily.

The 2 new marker classes still honor all the same styles as before and someone using this can continue to go a long way customizing them that way before they'd need to either subclass them or replace them. I'm not super happy with the code here but it's not too bad, but there may be some svg styles I've missed (like stroke-dasharray):

https://github.com/domoritz/leaflet-locatecontrol/pull/215/files#diff-351f3423206f4f74f9a9f72e1feae1afR58

Reply:
The circles seem a bit larger, though so I suggest we adopt that. The syroke width of the accuracy circle is also much thinner and I prefer that as well. I’m happy to make these style changes in a follow up but wanted to flag them here. 

The tunnel also shows the accuracy of the direction, which I think is nice. 

I also noticed that the white part is pulsing on iOS  but I won’t think we need that. I just haven’t noticed it before. 

On android, the circle does not pulse but also seems a bit larger.

Reply:
Why are you animating the accuracy circle, though? It seems quite distracting (albeit being beautiful).